README.md

```diff
@@ -1,3 +1,11 @@
 # Astarte Flow
 
 Astarte Flow lets you create reusable Blocks and Pipelines. This way, you can reuse the same pipeline in countless computation scenarios with different settings. And you can share your pipelines with others, too! Bundle your algorithm in a container and you’re ready to go.
+
+## Resources and Quickstart
+
+ * [Astarte Flow Documentation](https://docs.astarte-platform.org/flow/latest) - The main resource to learn about Astarte Flow.
```

Comment:
Just leave `/flow`, we'll handle the redirect from there (and right now there is no `latest` since we have no stable version yet)